src/aleph_vrf/coordinator/vrf.py

```diff
@@ -80,12 +92,20 @@ async def _generate_vrf(
 
     nonce = generate_nonce()
 
+    if request_id:
+        existing_message = await get_existing_vrf_message(aleph_client, request_id)
+        if existing_message:
+            message = PublishedVRFResponse.from_vrf_post_message(existing_message)
+            await check_message_integrity(aleph_client, message)
```

Comment:
Raised errors about unhonest nodes are not being returned in an informative response. As it looks right now, we will only return a 500 with Internal Server Error and nothing else.

Reply:
This is not true, if the coordinator is executed as VM, at the moment to handle the response, if some Exception have raised, we return an error, but we specify the error message (see here: https://github.com/aleph-im/aleph-vrf/blob/main/src/aleph_vrf/coordinator/main.py#L50).
And if we execute the coordinator as a library, it will raise the exception, so should be fine.